index.html

```diff
@@ -0,0 +1,66 @@
+    </head>
+    <body>
+        <header>
+            <div class="wrapper">
+                <nav class="nav-bar">
+                    <ul>
+                        <li class="logo"><a href="#">My Logo</a></li>
+                    </ul>
+                    <div class="menu">
+                            <div class="toogle-btn"></div>
+                            <div class="toogle-btn"></div>
+                            <div class="toogle-btn"></div>
+                    </div>
+                </nav>
+            </div>
+        </header>
+
+        <section>
+            <div class="headline-wrapper">
+                <div class="headline-container">
+                    <div class="intro-text">I’m Clinton<br> Glad to see you!</div>
+                    <div class="intro-message">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+                    <div class="connect">
+                        <h5>LET'S CONNECT</h5>
+                    </div>
+                </div>
+                
+                <div class="social">
+                    <div class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-medium"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-github"></i></a></div>
+                    <div class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></div>
+                </div>
+            </div>
+        </section>
+
+
+
+        <!-- <section>
+            <div class="headline-wrapper">
+                <div class="headline-text">
+                    <h1 class="intro-text">I’m Clinton<br> Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <h5>LET'S CONNECT</h5>
+                    <ul class="social">
+                        <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+                        <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                        <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+                        <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+                        <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+                    </ul>
+                </div>
+            </div>
+        </section> -->
```

Comment:
- [ ] Optional: Kindly remove all unnecessary comments in your code. Removing all unnecessary comments makes your code clean and professional.